recipes/supervised/meta.yaml

```diff
@@ -21,7 +20,6 @@ requirements:
     - pip
     - python >=3.7
   run:
-    - python ==3.7
```

Comment:
Putting `python >=3.7`  here would be the expected thing as it is defined in host.

Reply:
the host `- python >= 3.7` should be removed?

Reply:
You put the python constraint in both host and run

Reply:
where it should be defined? in the host only?

Reply:
In `host` and `run` with the same constraint.

Reply:
Thank you! Got it!